docs/clients/nft-client.md

```
@@ -52,6 +52,8 @@
 | `wipeNft(TokenId tokenId, long serialNumber, AccountId accountId, PrivateKey wipeKey)` | Wipes a single NFT using a custom wipe key. Returns total supply. |
 | `wipeNfts(TokenId tokenId, Set<Long> serialNumbers, AccountId accountId)` | Wipes multiple NFTs from an account using the operator wipe key. Returns total supply. |
 | `wipeNfts(TokenId tokenId, Set<Long> serialNumbers, AccountId accountId, PrivateKey wipeKey)` | Wipes multiple NFTs using a custom wipe key. Returns total supply. |
+| `pauseNfts(TokenId tokenId, PrivateKey pauseKey)` | pauses the NFT's from an account using the pause key.
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)